lib/dialects/sqlite/query.js

```diff
@@ -295,6 +302,50 @@ class Query extends AbstractQuery {
     });
   }
 
+  parseConstraintsFromSql(sql) {
+    let constraints = sql.split('CONSTRAINT ');
+    let referenceTableName, referenceTableKeys, update_action, delete_action;
+    constraints.splice(0, 1);
+    constraints = constraints.map(constraintSql => {
+      //Parse foreign key snippets
+      if (constraintSql.indexOf('REFERENCES') !== -1) {
+        //Parse out the constraint condition form sql string
+        update_action = constraintSql.match(/ON UPDATE (CASCADE|SET NULL|RESTRICT|NO ACTION|SET DEFAULT){1}/);
+        delete_action = constraintSql.match(/ON DELETE (CASCADE|SET NULL|RESTRICT|NO ACTION|SET DEFAULT){1}/);
```

Comment:
Postgres and MSSQL return format doesn't control variable name we use, does it ?

Just rename `update_action` => `updateAction` , `delete_action` => `deleteAction`